apps/mobile-wallet/src/features/send/screens/VerifyScreen.tsx

```diff
@@ -53,35 +47,25 @@ const VerifyScreen = ({ navigation, ...props }: ScreenProps) => {
       {...props}
     >
       <ScreenSection>
-        <Surface>
-          <Row title={t('Sending')} titleColor="secondary">
-            <AssetAmounts>
-              {assets.map(({ id, amount }) =>
-                amount ? <AssetAmountWithLogo key={id} assetId={id} amount={BigInt(amount)} fullPrecision bold /> : null
-              )}
-            </AssetAmounts>
-          </Row>
-
-          <TotalWorthRow assetAmounts={assetAmounts} />
-
-          <Row title={t('To')} titleColor="secondary">
-            <AddressBadge addressHash={toAddress} />
-          </Row>
-          <Row title={t('From')} titleColor="secondary">
-            <AddressBadge addressHash={fromAddress} />
-          </Row>
-          <Row title={t('Estimated fees')} titleColor="secondary" isLast>
-            <FeeAmounts fees={fees} />
-          </Row>
-        </Surface>
+        {chainedTxProps ? (
+          <>
+            <Surface type="accent" style={{ padding: 20 }}>
+              <AppText color="accent">
+                {t('Your address is missing some ALPH for gas fees but thankfully another address can cover it!')}
+              </AppText>
```

Comment:
Let's center the text, and as for the content, I'd explain what is being done, like:

```The origin address doesn’t have enough ALPH for gas fees. But don't you worry, the needed amount will be automatically transferred from another address in your wallet!``` WDYT?